src/Kiota.Builder/Writers/Java/JavaConventionService.cs

```diff
@@ -5,16 +5,16 @@
 namespace Kiota.Builder.Writers.Java {
     public class JavaConventionService : CommonLanguageConventionService
     {
-        private const string _streamTypeName = "InputStream";
-        public override string StreamTypeName => _streamTypeName;
-        private const string _voidTypeName = "Void";
-        public override string VoidTypeName => _voidTypeName;
+        private const string streamTypeName = "InputStream";
```

Comment:
This should start with a capital letter. Yes?
Same for line 11.

Reply:
thanks, I originally had not done it because of the right name would conflict with the interface property and having a non-constant would prevent me from using it in the PrimitiveTypes initalization. But _can simply use a different name 😁_